[Outlook] (new Outlook on Windows) Remove preview string (#4693)
OfficeDev/office-js-docs-pr: docs/outlook/outlook-on-send-addins.md

The on-send feature in Outlook requires that add-ins are configured for the send event types. Select the platform you'd like to configure.

# [Web browser (modern)/New Outlook on Windows](#tab/modern)

> Add-ins for Outlook on the web (modern) and new Outlook on Windows that use the on-send feature should run for any users who have them installed. However, if users are required to run on-send add-ins to meet compliance standards, then the mailbox policy must have the *OnSendAddinsEnabled* flag set to `true` so that editing the item is not allowed while the add-ins are processing on send.

Reviewer: ```suggestion
Add-ins for Outlook on the web (modern) and new Outlook on Windows that use the on-send feature should run for any users who have them installed. However, if users are required to run on-send add-ins to meet compliance standards, then the mailbox policy must have the *OnSendAddinsEnabled* flag set to `true` so that editing the item isn't allowed while the add-ins are processing on send.
```
